dist/index.html

```diff
@@ -15,13 +15,13 @@
         <h1 class='travel-title'>travel.</h1>
         <section class='login '>
           <input id='userName' class='user-name'type="text" name="username" value="" required>
-          <input id='password'class='password'type="text" name="password" value="" required>
+          <input id='password'class='password'type="password" name="password" value="" required>
```

Comment:
watch your spacing on these inputs. Notice there's no spaces between your tags on this line, but there are on line 17.

Also, I couldn't add the comment in Github on line 2, but for accessibility purposes it's good to add the LTR in your html tag.

Reply:
thank you - making the changes on my local repo!